### ISSUE TITLE
Properly paste system clipboard when search /?

### ISSUE THREAD

**Issue by tmpm697** [ggVGc/vim-fuzzysearch#5]
I map search function to use fuzzysearch:
```
g:fuzzysearch_prompt = 'fuzzy /'
g:fuzzysearch_hlsearch = 1
g:fuzzysearch_ignorecase = 1
g:fuzzysearch_max_history = 30
g:fuzzysearch_match_spaces = 0
```

Often I surf internet and copy text to system buffer then I want to paste to search it.
press `/` -> paste text

This plugin will not work properly and it will paste text directly to vim instead of under `/`
How to avoid it?

**Comment by ggVGc** [comment]
Yeah, this doesn't work for me either, and I'm not sure why right now.
But, when searching for something that you copied from somewhere else, I believe a normal search with `/` would be better anyway, rather than using FuzzySearch?
Personally, I don't replace my `/` with FuzzySearch, but use normal search when I want exact matches, and fuzzy search when I want it fuzzy.

**Comment by tmpm697** [comment]
I like fuzzy search to replace builtin search as it covers both exact math and regex match and I often don't exactly know what I'm looking for or too lazy to type the whole pattern. 

E.g search for: I need more time to think

I'll do fuzzy: need think

Also builtin doesn't support space.

I asked in vim stackexchange but seems no one reply.
